Replace JAEGER_IMAGE_TAG with JAEGER_VERSION

ADI-ROXX opened the pull request:
## Which problem is this PR solving?
- Resolves #6612 

## Description of the changes
- Replace JAEGER_IMAGE_TAG with JAEGER_VERSION in .yml files

## How was this change tested?
- 

## Checklist
- [ ] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [ ] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `npm run lint` and `npm run test`

yurishkuro commented on cmd/query/app/http_handler.go:
```diff
@@ -183,7 +183,7 @@ func (aH *APIHandler) transformOTLP(w http.ResponseWriter, r *http.Request) {
 
 	traces, err := otlp2traces(body)
-	if aH.handleError(w, err, http.StatusInternalServerError) {
+	if aH.handleError(w, err, http.StatusBadRequest) {
```
why is this in this PR?

ADI-ROXX replied:
I'll fix it right away.

ADI-ROXX commented:
@yurishkuro The issue in this pr is because of a commit that I have added to my main branch for another commit. I have fixed the main branch that I have forked. I have added a new pr #6614 that resolves this issue.
